tests/integration_tests/build/test_doctests.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""A test that ensures that all unit tests pass at integration time.
+
+# TODO
+
+- Run with `--release` once  `https://github.com/edef1c/libfringe/issues/75`
+  is fixed
+"""
+
+import platform
+
+import pytest
+
+import host_tools.cargo_build as host  # pylint:disable=import-error
+
+MACHINE = platform.machine()
+TARGETS = ["{}-unknown-linux-gnu".format(MACHINE),
+           "{}-unknown-linux-musl".format(MACHINE)]
+
+
+@pytest.mark.parametrize(
+    "target",
+    TARGETS
+)
+@pytest.mark.skipif(
+    platform.machine() != "x86_64",
+    reason="Doctests are already tested by test_unittest.py on aarch64"
+)
+def test_doctests(test_session_root_path, target):
+    """Run doc tests for all supported targets."""
+    extra_env = ''
+    extra_args = "--target {} --doc --exclude cpuid".format(target)
```

Comment:
As far as I know, --doc will not work with musl target, it simply ignores it. It will just run unit tests. Can you please check that the doc tests are indeed run when musl target is used?

Reply:
I confirm that 
`RUST_TEST_THREADS=1 sudo env "PATH=$PATH" cargo test --all --no-fail-fast --target aarch64-unknown-linux-musl  --exclude cpuid --doc
    Finished dev [unoptimized + debuginfo] target(s) in 0.23s
`
will not run doc-tests, so my suggestions is to remove the target parameter and hardcode it to `gnu`.

Reply:
Okay, I will do this